test/test_integration_policy.py

```diff
@@ -0,0 +1,316 @@
+        Utils.load_policy_from_string(self, policy)
+
+        for variable in variables:
+            Utils.assert_set_and_get(self, variable)
+
+    @integration_test
+    def test_https_load_policy_can_output_returned_data(self):
+        self.setup_cli_params({})
+
+        user_id1 = uuid.uuid4().hex
+        user_id2 = uuid.uuid4().hex
+        policy = "- !user {user_id1}\n- !user {user_id2}\n".format(user_id1=user_id1,
+                                                                   user_id2=user_id2)
+
+        # Run the new load that should not result in newly created roles
+        json_result = json.loads(Utils.load_policy_from_string(self, policy))
+
+        account_name = self.client_params.account
+        expected_object = {
+            'version': json_result['version'],
+            'created_roles': {
+                f'{account_name}:user:' + user_id1: {
+                    'id': f'{account_name}:user:' + user_id1,
+                    'api_key': json_result['created_roles'][f'{account_name}:user:' + user_id1]['api_key'],
+                },
+                f'{account_name}:user:' + user_id2: {
+                    'id': f'{account_name}:user:' + user_id2,
+                    'api_key': json_result['created_roles'][f'{account_name}:user:' + user_id2]['api_key'],
+                }
+            }
+        }
+
+        self.assertDictEqual(json_result, expected_object)
+
+    '''
+    Validates that a policy command without a subcommand 'load'
+    in this case, will fail and return help screen
+    '''
+    @integration_test
+    def test_policy_load_without_subcommand_returns_help_screen(self):
+        with redirect_stderr(self.capture_stream):
+            self.invoke_cli(self.cli_auth_params,
+                                     ['policy', '-b', 'root', '-f', 'somepolicy.yml'], exit_code=1)
+        self.assertIn("Error argument action: invalid choice: 'root' (choose from", self.capture_stream.getvalue())
+
+    '''
+    Validates that a policy command without the policy path
+    will fail and return help screen
+    '''
+    @integration_test
+    def test_policy_load_without_policy_path_returns_help_screen(self):
+        with redirect_stderr(self.capture_stream):
+            output = self.invoke_cli(self.cli_auth_params,
+                           ['policy', '-b', 'root'], exit_code=1)
+        self.assertIn("Error argument action: invalid choice: 'root' (choose from", self.capture_stream.getvalue())
+
+    '''
+    A non-existent policy file will return FileNotFound error message
+    '''
+    @integration_test
+    def test_policy_load_raises_file_not_exists_error(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['policy', 'load', '-b', 'root', '-f', 'somepolicy.yml'], exit_code=1)
+        self.assertRegex(output, "Error: No such file or directory:")
+
+    '''
+    A non-existent policy file will return FileNotFound error message
```

Comment:
This is a bad syntax test

Reply:
that reminded me we also want to check the debug logs for the proper feedback. For example:
```
422 Unprocessable Entity {"error":{"code":"validation_failed","message":"policy_text Error at line 1, column 0 in  : Unrecognized data type '!'","details":[{"code":"validation_failed","target":"policy_text","message":"Error at line 1, column 0 in  : Unrecognized data type '!'"}]}}
```